virttest/utils_misc.py

```diff
@@ -1699,6 +1701,19 @@ def read_from_node_meminfo(self, node_id, key):
         """
         return self.get_all_node_meminfo()[node_id][key]
 
+    def get_all_node_cpus(self):
+        """
+        Get the cpus of all online nodes.
+
+        :return: All nodes' cpus
+        :rtype: Dict
+        """
+        cpus = {}
+        node = NumaNode(session=self.session)
+        for node_id in self.get_online_nodes():
+            cpus[node_id] = node.get_node_cpus(node_id)
```

Comment:
This function will run 'numactl -H'. To get all node cpus info, we only need run it one time. I suggest either not to call get_nodes_cpus() or update it.

Reply:
Then it go back to the initial version of the tp-qemu one, but just move part of them from tp-qemu to vt...

I am sure i will not update the class NumaNode which effect many other test while has little to do with this test.
Update this function to get the all the numa cpus without call `get_nodes_cpus()` is another kind of duplication.
Base on the current design we can not avoid all repeated call, which one is acceptable for you ?

Reply:
I choose rewrite function `get_all_node_cpus`, do not call `get_nodes_cpus()`. 
Both `numactl -H` and `cpulist` can achieve the purpose and i choose the second one, please help review again, thanks!